cmd/newtemplates/protocol.js

```diff
@@ -0,0 +1,52 @@
+import http from "k6/http";
+import { check, sleep } from "k6";
+
+const BASE_URL = __ENV.BASE_URL || 'https://quickpizza.grafana.com';
+
+export const options = {
+  vus: 10,
+  stages: [
+    { duration: "10s", target: 5 },
+    { duration: "20s", target: 10 },
+    { duration: "1s", target: 0 },
+  ], thresholds: {
+    http_req_failed: ["rate<0.01"],
+    http_req_duration: ["p(95)<500", "p(99)<1000"],
+  },{{ if .EnableCloud }}
+  cloud: { {{ if .ProjectID }}
+    projectID: {{ .ProjectID }}, {{ else }}
+    // projectID: 12345, // Replace this with your own projectID {{ end }}
+    name: "{{ .ScriptName }}",
+  }, {{ end }}
+};
+
+export function setup() {
+  let res = http.get(BASE_URL);
+  if (res.status !== 200) {
+    throw new Error(
+      `Got unexpected status code ${res.status} when trying to setup. Exiting.`
+    );
+  }
+}
+
+export default function() {
+  let restrictions = {
+    maxCaloriesPerSlice: 500,
+    mustBeVegetarian: false,
+    excludedIngredients: ["pepperoni"],
+    excludedTools: ["knife"],
+    maxNumberOfToppings: 6,
+    minNumberOfToppings: 2
+  };
+
+  let res = http.post(BASE_URL + "/api/pizza", JSON.stringify(restrictions), {
+    headers: {
+      'Content-Type': 'application/json',
+      'X-User-ID': 23423,
```

Comment:
The new versions of QuickPizza no longer use this approach to Auth. 

This needs to be fixed, and the public QuickPizza service needs to be redeployed. cc @federicotdn for visibility.

Reply:
Fixed.